flavors.gradle

```diff
@@ -0,0 +1,21 @@
+ext.flavors = {
+
+    flavorDimensions FLAVORS_DIMENSION
+
+    productFlavors {
```

Comment:
is this flavor confirmed with BE team ? bcs i think if we need to add some flavors, we need to compare it with BE flavors too

Reply:
development
preproduction
production

Reply:
noted  👍